src/main/java/io/cdap/plugin/gcp/gcs/StorageClient.java

```diff
@@ -301,6 +301,16 @@ public static StorageClient create(String project, @Nullable String serviceAccou
     return new StorageClient(storage);
   }
 
+  public static StorageClient create(String project, @Nullable String serviceAccount,
```

Comment:
Is `create(String project, @Nullable String serviceAccountPath)` (L295) still needed?

Reply:
@rmstar No, we have removed in new PR.